atomic-git/f23_spc/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# based on Dockerfile by Brent Baude
+
+FROM openscap-daemon-f23
+
+RUN dnf -y update && dnf -y groupinstall "Development Tools" && dnf -y install make wget git && dnf clean all
+
+RUN dnf -y install 'dnf-command(builddep)' && dnf builddep -y atomic openscap-daemon openscap && dnf -y install libtool && dnf clean all
+
+RUN git clone -b maint-1.2 https://github.com/OpenSCAP/openscap.git && \
+    pushd /openscap && ./autogen.sh && \
+    ./configure --enable-sce --prefix=/usr && make -j 4 install && popd
+
+RUN git clone https://github.com/OpenSCAP/openscap-daemon && \
+    pushd /openscap-daemon && python3 setup.py install && popd
+
+RUN dnf -y install python3-requests && git clone https://github.com/projectatomic/atomic.git && \
```

Comment:
python3-requests should be already installed by `dnf builddep -y atomic` on line 7